Doc/tools/templates/layout.html

```diff
@@ -1,5 +1,14 @@
 {% extends "!layout.html" %}
 
+{% block header %}
+{%- if outdated_message is defined and outdated_link_text is defined %}
+<div id="outdated-warning" style="padding: .5em; text-align: center; background-color: #FFBABA; color: #6A0E0E;">
+    {{ outdated_message }}
+    <a href="/3/{{ pagename }}{{ file_suffix }}">{{ outdated_link_text }}</a>.
```

Comment:
I would just make a flag to *show* the outdated message, and then just have the message itself and the link text be static text wrapped in `{% trans %}` to ease translation.

Reply:
translations? I like translations!

Reply:
Good for the heart, good for the soul. :)

Reply:
It would be nice to have two translations strings:

> This document is for an old version of Python that is no longer supported. You should upgrade, and read the {link}.

and

> Python documentation for the last stable release

Using the current tags I'll get:

> This document is for an old version of Python that is no longer supported. You should upgrade, and read the

and

> Python documentation for the last stable release

Almost the same, but in the current version translations can't move the link inside the sentence, and they can't touch the trailing dot.